src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -205,96 +205,91 @@
     type: command
     parameters:
         - name: --ids
-          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
+          short-summary: "One or more VM scale sets or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
 """
 
 helps['vmss reimage'] = """
     type: command
     parameters:
         - name: --ids
-          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
+          short-summary: "One or more VM scale sets or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
 """
 
 helps['vmss show'] = """
     type: command
     parameters:
         - name: --ids
-          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
+          short-summary: "One or more VM scale sets or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
 """
 
 helps['vm convert'] = """
     type: command
-    short-summary: Convert VM with unmanaged disks to use managed disks
+    short-summary: Convert a VM with unmanaged disks to use managed disks.
     examples:
-        - name: Convert VM with unmanaged disks to managed by resource group and name
+        - name: Use resource group and name to convert a VM with unmanaged disks to use managed disks.
           text: az vm convert -g group_name -n vm-name
 {0}
 """.format(vm_ids_example.format('Convert VM with unmanaged disks to managed by Ids', 'vm convert'))
 
 helps['vm'] = """
     type: group
-    short-summary: Provision Linux and Windows virtual machines in minutes
+    short-summary: Provision Linux or Windows virtual machines in minutes.
 """
 helps['vm user'] = """
     type: group
-    short-summary: Manage users
+    short-summary: Manage a user account on a VM.
 """
 
 helps['vm user delete'] = """
     type: command
     long-summary: >
-        Delete a user account without logging into to the VM directly.
+        Delete a user account from a VM without logging into it.
```

Comment:
I believe these only work for Linux VMs. If so, we should call that out in the help. @yugangw-msft can you confirm?